src/Button/OutlineButton.js

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import Button from './Button'
+
+const OutlineButton = styled(Button)`
+  background-color: ${props => props.theme.colors.white};
```

Comment:
I'd go with `background-color: transparent` here for support on non-white backgrounds